src/app/nav-bar/nav-bar.component.css

```diff
@@ -0,0 +1,37 @@
+
+* {
+  margin: 0;
+  padding: 0;
+
+  font-family: Georgia;
+}
+body {
+  margin: 0;
+  padding: 0;
+}
+
+.wrapper {
+  width: 90%;
+
+  margin: 0 auto;
+}
+
+#Navigation {
+  position: fixed;
+  top: 0;
+  width: 100%;
+  height: 100px;
+  background: url(C:\Users\fcelta\WebstormProjects\AlorFE\src\assets\img\what-the-hex.png);
+}
+
+#logo {
+  width: 60px;
+  height: 60px;
+
+  float: left;
+
+  margin: 20px;
+  background: url(C:\Users\fcelta\WebstormProjects\AlorFE\src\assets\img\coffeeright250x250.png);
```

Comment:
the code crashes on other devices:
```
./src/app/nav-bar/nav-bar.component.css
Module Error (from ./node_modules/postcss-loader/src/index.js):
(Emitted value instead of an instance of Error) CssSyntaxError: C:\Users\aneuhaeuser\Documents\Onbordingphase\Alor-Frontend\src\app\nav-bar\nav-bar.component.css:34:19: Can't resolve '/Users/fcelta/WebstormProjects/AlorFE/src/assets/img/coffeeright250x250.png' in 'C:\Users\aneuhaeuser\Documents\Onbordingphase\Alor-Frontend\src\app\nav-bar'

  32 | 
  33 |   margin: 20px;
> 34 |   background: url(C:\Users\fcelta\WebstormProjects\AlorFE\src\assets\img\coffeeright250x250.png);
     |                   ^
  35 |   background-size: cover;
  36 | }
```